_posts/2023-12-06-#2760.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: pr
+date: 2023-00-00
+title: "Fix infinite loop when closing a pre-funding channel"
+pr: 2760
+authors: [jbesraa]
+components: ["channel-establishment", "channel-funding"]
+host: jbesraa
+status: upcoming
+commit:
+---
+
+## Notes
+  Creating a channel involves exchanging multiple messages with a
+  connected peer, which does not always end up successfully.
+  Specificly, in the current implemntation an infinite loop might occur 
+  in the funding step.
```

Comment:
I think we should mention that this bug resulted from https://github.com/lightningdevkit/rust-lightning/pull/2077 specifically.